client/Windows/wf_client.c

```diff
@@ -537,6 +537,8 @@ BOOL wf_authenticate(freerdp* instance, char** username, char** password, char**
 
 	if (strlen(Domain) > 0)
 		*domain = _strdup(Domain);
+	else
+		*domain = _strdup("\0");
 
```

Comment:
The first is not your code but those strdups should be checked (for the original code this is done in PR #2708)